modules/govuk_mysql/templates/s3cfg.erb

```diff
@@ -0,0 +1,3 @@
+[default]
+bucket_location = eu-west-1
```

Comment:
Should this be dynamic? The define allows it to be set to regions other than eu-west-1